### PR TITLE
Natlab Police: Xfail failed nigthly tests

### PR DESCRIPTION
**Pull request by arvydasv**
### :ballot_box_with_check: Definition of Done checklist
- [x] Commit history is clean ([requirements](../../blob/main/docs/git_commit_messages_requirements.md))
- [x] README.md is updated
- [x] changelog.md is updated

### CODE REVIEW

**Review by gytsto — DISMISSED** [review]
+2

File: nat-lab/tests/test_direct_connection.py [hunk 1]
```diff
@@ -591,6 +592,7 @@ async def test_direct_connection_loss_for_infinity(
             ConnectionTag.DOCKER_CONE_CLIENT_2,
             "upnp",
             "stun",
+            marks=pytest.mark.xfail(reason="test is flaky - LLT-4115"),
```
Comment (gytsto) [inline nat-lab/tests/test_direct_connection.py:595]: just mark this entire test as flaky

Comment (gytsto) [reply nat-lab/tests/test_direct_connection.py:595]: got failure on other parameter set on my branch

**Review by gytsto — APPROVED** [review]

**Review by Jauler — APPROVED** [review]
+2.0

**Pull request merged**